hadoop-client-modules/hadoop-client-check-invariants/pom.xml

```diff
@@ -85,7 +85,7 @@
                     <!-- Leave commons-logging unshaded so downstream users can configure logging. -->
                     <exclude>commons-logging:commons-logging</exclude>
                     <!-- Leave log4j unshaded so downstream users can configure logging. -->
```

Comment:
nit: comment needs updating

Reply:
I updated the comments. Thanks, @steveloughran.